test/letters/reducers/index.unit.spec.js

```diff
@@ -184,22 +217,43 @@ describe('letters reducer', () => {
     );
 
     expect(state.countries).to.contain('USA');
+    expect(state.countriesAvailable).to.be.true;
```

Comment:
I think this is bordering on :paranoid-parrot: but it seems that we're only testing a specific value of the new state in general. Wouldn't it be more comprehensive that the whole new state is `eql` to what we expect? That way, we're not inadvertently letting other state updates or mutations get through our tests. E.g., if one of the reducers is changed down the road to update the original thing *and* something else, it'll fail our tests until the test is updated appropriately.

Reply:
I agree with the :paranoid-parrot:, but I like it. Should be a pretty easy update.